src/CommitmentsV2/SFA.DAS.CommitmentsV2/Services/ResolveOverlappingTrainingDateRequestService.cs

```diff
@@ -93,6 +92,10 @@ private async Task<bool> CheckCanResolveOverlap(OverlappingTrainingDateRequest o
                 // resolve overlap if any of the mandatory fields missing
                 return true;
             }
+
+            if (resolutionType == OverlappingTrainingDateRequestResolutionType.ApprentieshipIsStillActive)
+                return true; // resolve if employer has confirmed that the apprenticeship Is Stil lActive
```

Comment:
please add a log like in  other if blocks

Reply:
Done